NaReguaApp/App.js

```diff
@@ -1,18 +1,12 @@
-/**
- * Sample React Native App
- * https://github.com/facebook/react-native
- *
- * @format
- * @flow strict-local
- */
-
 import React from 'react';
-import Route from './src/routes/route';
+import Routes from './src/routes/index';
+import { NavigationContainer } from '@react-navigation/native'
```

Comment:
Codacy has a fix for the issue: [Replace `·NavigationContainer·}·from·'@react-navigation/native'` with `NavigationContainer}·from·'@react-navigation/native';`](https://app.codacy.com/gh/UFAPE-Projetao20202/naregua_front/pullRequest?prid=8386487)
```suggestion
import {NavigationContainer} from '@react-navigation/native';
```